src/plopp/backends/manager.py

```diff
@@ -24,6 +26,10 @@ def values(self):
     def items(self):
         return self._mapping.items()
 
+    def reset(self):
+        self._mapping = DEFAULTS.copy()
```

Comment:
Why does it store `DEFAULTS` as an instance variable? And why does it reset it here?

Reply:
I'm not sure I understand the question. Resetting here inside the `reset` method makes sense?
Are you saying I should do it differently, something like
```Py
self._mapping.clear()
self._mapping.update(DEFAULTS)
```
?

Reply:
Ah sorry. I got confused about `_mapping` and `_backends` and thought that `_mapping` never changes.